aws/cloudfront_distribution_configuration_structure.go

```diff
@@ -223,13 +223,28 @@ func expandCloudFrontDefaultCacheBehavior(m map[string]interface{}) *cloudfront.
 }
 
 func expandCacheBehavior(m map[string]interface{}) *cloudfront.CacheBehavior {
+	var forwardedValues *cloudfront.ForwardedValues
+	if forwardedValuesFlat, ok := m["forwarded_values"].([]interface{}); ok && len(forwardedValuesFlat) == 1 {
+		forwardedValues = expandForwardedValues(m["forwarded_values"].([]interface{})[0].(map[string]interface{}))
+	}
```

Comment:
It does. That looks much better.

Reply:
I actually put this one back, because I wanted to try and keep `expandForwardedValues` accepting a map like the other expand functions. I had to do the check that the map exists outside of the function in order to do that.